### PR TITLE
Add new file test.html

### PR DESCRIPTION
**Pull request by VanGiang**

### CODE REVIEW

**Review by DinhNgocDo — COMMENTED** [review]
ok a

**Review by TranKienGiang — APPROVED** [review]
very good

File: test.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+    PHP2104
```
Comment (pham-van-thang) [inline test.html]: thiếu thẻ h1.

File: test.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+</body>
+</html>
```
Comment (huyd12vt1) [inline test.html]: thieu the h1 a oi =)))

**Review by pham-van-thang — APPROVED** [review]

**Pull request merged**